### PR TITLE
Czech translation (cs)

### PR DESCRIPTION
**Pull request by 447937**
Translation for czech language.

### CODE REVIEW

**Comment by marius-wieschollek** [comment]
That looks cool so far. I will take a closer look at it tomorrow.

**Comment by 447937** [comment]
Thanks, I hope that everything will be ok.

File: src/l10n/cs.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+            'Edit'                                : 'Upravit',
+            'Rename'                              : 'Přejmenovat',
+            'Delete'                              : 'Smazat',
+            'Restore'                             : 'Obnovit',
+            'New Password'                        : 'Nové heslo',
+            'New Folder'                          : 'Nová složka',
+            'New Tag'                             : 'Nová značka',
+            'Secure'                              : 'Bezpečné',
+            'Weak'                                : 'Slabé',
+            'Broken'                              : 'Velmi slabé',
```
Comment (marius-wieschollek) [inline src/l10n/cs.js]: If i'm correct, "Velmi slabé" translates to "Very weak". Is there a stronger alternative for this? The idea is to tell the user that passwords in this category are so bad, you can't consider them to keep anyone from breaking them.

Comment (447937) [reply src/l10n/cs.js]: Yes, you are correct "Velmi slabé" translates as "Very weak". "Broken" could be translated as "Rozbité" without context - but that would not make sense here. I think that in this situation "Broken" should be translated as "Nebezpečné" (dangerous) or "Velmi nebezpečné" (very dangerous). Should I change this in my repository and create new pull request? I'm new to github, so I'm sorry for this question.

Comment (marius-wieschollek) [reply src/l10n/cs.js]: Great. You can just change it in your repository and github will update the pull request as soon as you push the changes.

Comment (447937) [reply src/l10n/cs.js]: OK, done.

**Review by marius-wieschollek — APPROVED** [review]
Looks Good

**Comment by marius-wieschollek** [comment]
Done

**Pull request merged**